configuring/configuring-healthwatch.html.md.erb

```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
 
 1. Click **Save**.
 
+## <a id='resource-config'></a> (Optional) Configure for OpenTelemetry
+
+Follow these steps to configure HealthWatch and receive data through the OpenTelemetry collector for Tanzu Application Service 6.0 and above:
+
+1. Click the Settings tab and select **Resource Config**.
```

Comment:
Click on Healthwatch exporter tile and select **Resource Config** under the Settings tab.